spec/q-spec.js

```diff
@@ -1192,7 +1192,7 @@ describe("any", function() {
         var deferreds = [Q.defer(), Q.defer()];
         var promises = [deferreds[0].promise, deferreds[1].promise];
 
-        return testReject(promises, deferreds);
+        return testReject(promises, deferreds, new Error('Rejected'));
```

Comment:
This line should not pass lint, since the quotes are inconsistent with the rest of the module.